schema/example.go

```diff
@@ -0,0 +1,173 @@
+package schema
+
+func NewExample() Schema {
+	location := locationSchema{
+		Lat:  35.712758,
+		Long: 51.392114,
+	}
+
+	score := scoreSchema{
+		Type:  "GPA",
+		Value: "3.4",
+	}
+
+	resource := resourceSchema{
+		URL:   "http://www.example.com/my-example-slides/",
+		Label: "Slides",
+	}
+
+	core := coreSchema{
+		Name:              "John Doe",
+		Title:             "Software Engineer",
+		Image:             "example.com/Abcxyz",
+		Email:             "lucas@example.com",
+		Phone:             "http://www.example.com/my-slides/",
```

Comment:
Value seems incorrect.

Reply:
Fixed in 570748d476f068d3be54b49ed54ae44398a2eeaa